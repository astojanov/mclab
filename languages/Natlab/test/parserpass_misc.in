a = M'+M'
@handle
global x y
persistent x;
return