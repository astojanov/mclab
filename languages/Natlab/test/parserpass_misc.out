1 1
5 6
a = ((M') + (M'))
@handle
global x y
persistent x;
return
